bin/update_logger_image.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# ---
+# IMPORTANT: The "cwm-worker-logger.image" is updated automatically via CI.
+#            DO NOT update it manually unless you know what you're doing!
+# ---
+
+LOGGER_IMAGE_FILE="./helm/cwm-worker-logger.image"
+VALUES_FILE="./helm/values.yaml"
+if test -f "$LOGGER_IMAGE_FILE"; then
+  if grep -Fq -- "metricsLogger" $VALUES_FILE; then
+    sed -i "s/^\([[:space:]]\+image:\).*/\1 $(cat $LOGGER_IMAGE_FILE)/" $VALUES_FILE
```

Comment:
this replaces both the minio and the logger images (and possible future image values we may have in the helm values)

please use a simpler sed to just replace the logger image - 

```
sed -i "s#docker.pkg.github.com/cloudwebmanage/cwm-worker-logger/cwm-worker-logger#$(cat $LOGGER_IMAGE_FILE)#" $VALUES_FILE
```

Reply:
Tested and updated. Earlier, I also tried a proper YAML parser with a Ruby script. But, that seemed too much for this single line. Maybe, for a more intensive manipulation of YAML files, we can resort to that in the future.